src/Phinx/Config/Config.php

```diff
@@ -455,6 +455,7 @@ protected function recurseArrayForTokens($arr, $tokens)
             }
             if (is_string($value)) {
                 foreach ($tokens as $token => $tval) {
+                    $tval = $tval === null ? '' : $tval;
                     $value = str_replace($token, $tval, $value);
```

Comment:
Could simplify with a null coalescing within where we use `$tval`.

```suggestion
                    $value = str_replace($token, $tval ?? '', $value);
```